pcdsdevices/analog_signals.py

```diff
@@ -61,6 +61,56 @@ class Acromag(BaseInterface, Device):
     tab_component_names = True
 
 
+class AcromagChannelInput(BaseInterface, Device):
+    """
+    Class for an Acromag Channel Input.
+
+    Parameters
+    ----------
+    prefix : str
+        The base EPICS PV for the Acromag. E.g.: `XPP:USR:ai1`
+    channel : str
+        The channel number. E.g.: `1` [0-15]
+    name : str
+        A name to refer to the Acromag Input Channel.
+    """
+    channel = FCpt(EpicsSignalRO, '{self._prefix}:{self._channel}',
+                   kind='normal', doc='Analog Input [Volts]')
+
+    tab_component_names = True
+
+    def __init__(self, prefix, channel, **kwargs):
+        self._channel = channel
+        self._prefix = prefix
+        self._name = ''.join(['ai1_', self._channel])
```

Comment:
Will it always be 1? Same with output

Reply:
Also, maybe it could be better to accept a name in the args

Reply:
Yeah this is a great point - thank you, I don't think I need to figure out the name here at all, I'm just going to give it a default name when initializing it.  👍

Reply:
changed it to `ai_` and `ao_`   in 1381498 - hopefully this is fine

Reply:
Yeah that should be good. I've just added a new suggestion for your consideration, adding the possibility to specify a name instead of taking the default, allowing the user to name it according to the channel's purpose.